.github/workflows/testpypi-publish-kensu-pyspark-on-pr.yml

```diff
@@ -0,0 +1,54 @@
+name: Test Kensu Python 🐍 Agent on PRs
+
+on:
+  pull_request:
+    # when target is main
+    branches: [ main ]
+  push:
+    branches: [ main ]
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [ 3.7 ]
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install -e ".[all]"
+    - name: Test with pytest
+      run: |
+        CONF_FILE=tests/unit/conf.ini pytest --log-cli-level=INFO
+    - name: Install pypa/build
+      run: >-
+        python -m
+        pip install
+        build
+        --user
+    - name: Store unix timestamp in BUILD_NUMBER env var
+      run: echo "BUILD_NUMBER=.dev$(date +%s)" >> $GITHUB_ENV
+    - name: Build a binary wheel and a source tarball
+      # store PR number in BUILD_FLAVOR env var
+      # so we'll get version like 0.0.1.123.dev1694786623 where 123 is PR number
+      env:
+        BUILD_FLAVOR: .alpha${{ github.event.number }}
+      run: >-
+        DELIVERY_ARTIFACT_NAME="kensu-pyspark" python -m
```

Comment:
only difference from regular testpypi is extra `DELIVERY_ARTIFACT_NAME="kensu-pyspark"` in this line